### DISCUSSION TITLE
Can't get my Event Update to email my contributors like it did a few months ago

### DISCUSSION THREAD

**Discussion by majora2007** [opencollective/opencollective#7564] [Q&A] [answered]
I can't find any documentation about this online, why it wouldn't be emailing, but I selected public and Contributors, but when I got to publish, it seems like an email will not go out?

I want to use the same flow I've been using for years. I want my email to go out to those who support my application. How can I do that? 

![image](https://github.com/user-attachments/assets/95fdcb34-aae7-4ff8-a31c-7e05b80b7e0d)

**Comment by Betree** [comment]
Hi @majora2007, sorry for the late reply - we're not monitoring discussions as closely as issues. Can you share your collective URL?

**Reply by majora2007** [reply]
https://opencollective.com/kavita

Do note, that despite this warning, I did hit Publish and emails were sent. Seems to be a bug in the UI. I have yet to test it again after this issue report.

**Reply by Betree** [reply, marked as answer]
Thanks! You probably faced a network error, which wrongly translates in the interface as "Your update will not be sent to anyone". We'll change that in https://github.com/opencollective/opencollective-frontend/pull/10697 to make the error explicit.

**Reply by majora2007** [reply]
I see, please let me know once it's deployed and on the next release of my software I can confirm.

**Reply by Betree** [reply]
It's live

**Reply by majora2007** [reply]
Thanks, I have a release coming up next month. I'll circle back then.

**Comment by majora2007** [comment]
Release completed, email went out and UI reflected how it should. Thanks.
